includes/wc-template-functions.php

```diff
@@ -1677,6 +1677,7 @@ function woocommerce_quantity_input( $args = array(), $product = null, $echo = t
 			'pattern'      => apply_filters( 'woocommerce_quantity_input_pattern', has_filter( 'woocommerce_stock_amount', 'intval' ) ? '[0-9]*' : '' ),
 			'inputmode'    => apply_filters( 'woocommerce_quantity_input_inputmode', has_filter( 'woocommerce_stock_amount', 'intval' ) ? 'numeric' : '' ),
 			'product_name' => $product ? $product->get_title() : '',
+			'placeholder'  => '',
```

Comment:
Could you please add a new filter as per the attributes above this, just so it follows the same trend.